sass/directives/_carousels.scss

```diff
@@ -88,6 +88,66 @@
   }
 }
 
+@mixin resource-header {
+  h2, p, h3, h1 {
+    display: inline-block;
+  }
+
+  nav {
+    display: inline-block;
+    float: right;
+  }
+}
+
+@mixin resource-carousel-arrow($arrow-z-index: 1) {
+  z-index: $arrow-z-index;
+  color: $inactive-carousel-button-color;
+  font-size: $direction-arrow-size / 2;
+  cursor: pointer;
+  border: 1px solid $inactive-carousel-button-color;
+  padding: $base-padding / 2;
+
+  &:hover,
+  &:active,
+  &:focus {
+    color: $active-carousel-button-color;
+    border: 1px solid $active-carousel-button-color;
+  }
+
+  &--left {
+    margin-right: 1px;
+  }
+}
+
+@mixin resource-continue-block {
```

Comment:
not sure what line this should correspond to but on smaller screens make sure the height of the continue block doesn't change, or set the padding on the block to be taller for small screens. This is happening at 661px

![image](https://cloud.githubusercontent.com/assets/2359538/23523092/1ed67ee0-ff4b-11e6-931d-955fc038eda8.png)